src/Featurizers/UnitTests/NormalizeFeaturizer_UnitTests.cpp

```diff
@@ -0,0 +1,41 @@
+// ----------------------------------------------------------------------

```

Comment:
I expect to see more tests here - at least 1 test for every exception thrown. There seems to be more functionality that this should exercise as well.

Reply:
Can you add a test for no inputs and inputs that are all null.

---
In reply to: [368049067](https://github.com/microsoft/FeaturizersLibrary/pull/72#discussion_r368049067) [](ancestors = 368049067)